daemon/src/projection.rs

```diff
@@ -962,6 +975,12 @@ mod round_to_two_dp {
         }
     }
 
+    impl ToDecimal for Decimal {
```

Comment:
😄

Reply:
Needed that to make the serde module work :D